fix: sambanova dropped a model support

[55](https://github.com/mozilla-ai/any-llm/actions/runs/23746112380/job/69175113472#step:5:1856)
=========================== short test summary info ============================
FAILED tests/integration/test_reasoning.py::test_completion_reasoning[sambanova] - openai.APIStatusError: The requested model (DeepSeek-R1-Distill-Llama-70B) is not available on SambaNova Cloud. For more information, please visit https://docs.sambanova.ai/cloud/docs/resources/deprecations. For further inquiries, please contact our support team at help@sambanova.ai.
FAILED tests/integration/test_reasoning.py::test_completion_reasoning_streaming[sambanova] - openai.APIStatusError: The requested model (DeepSeek-R1-Distill-Llama-70B) is not available on SambaNova Cloud. For more information, please visit https://docs.sambanova.ai/cloud/docs/resources/deprecations. For further inquiries, please contact our support team at help@sambanova.ai.
= 2 failed, 339 passed, 315 skipped, 64 warnings, 17 rerun in 304.22s (0:05:04) =
Error: Process completed with exit code 1.

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -35,7 +35,7 @@ def provider_reasoning_model_map() -> dict[LLMProvider, str]:
         LLMProvider.BEDROCK: "us.anthropic.claude-haiku-4-5-20251001-v1:0",
         LLMProvider.HUGGINGFACE: "Qwen/Qwen2.5-72B-Instruct",
         LLMProvider.NEBIUS: "openai/gpt-oss-20b",
-        LLMProvider.SAMBANOVA: "DeepSeek-R1-Distill-Llama-70B",
+        LLMProvider.SAMBANOVA: "DeepSeek-R1-0528",
         LLMProvider.TOGETHER: "openai/gpt-oss-20b",
         LLMProvider.PORTKEY: "@nebius-any-llm/Qwen/Qwen3-32B",
         LLMProvider.MINIMAX: "MiniMax-M2",

